changelogs/unreleased/fix_release_command.yml

```diff
@@ -0,0 +1,6 @@
+description: Ensure the `inmanta module release` command never decreases the version number
```

Comment:
This sounds more ominous than it is, which might not be ideal if we use it as a changelog section?

Reply:
changed